fastlane/Fastfile

```diff
@@ -38,18 +38,29 @@ platform :ios do
   end
 
   desc "Runs all the tests"
-  lane :test do
+  lane :test do |options|
+    wait_for_simulator = options.fetch(:wait_for_simulator, false)
+    if wait_for_simulator
+      puts 'sleeping for 5 minutes so the simulators have time to boot up. The sadness.'
+      sleep(5 * 60)
+      puts '5 minutes elapsed. Time to get back to work'
+    end
+    
```

Comment:
😢

Reply:
Hopefully we don't have to keep this for too long :)